src/test/link-rewriter.test.js

```diff
@@ -49,29 +49,29 @@ test.serial('rewriteLinks updates module link to topic', async t => {
 			});
 		}
 	};
-
-	const rewriter = new LinkRewriter('/testdata', MockValenceApi, fs);
+	const fileHandler = new FileHandler('/testdata', mockFs);
+	const rewriter = new LinkRewriter(fileHandler, MockValenceApi);
 
 	await rewriter.rewriteLinks('https://example.com/', OrgUnit, manifest);
 });
 
 test.serial('rewriteLinks updates topic link to module', async t => {
 	mockFs({
 		'/testdata/module1/index.html': '<html><body><p>Content</p></body></html>',
-		'/testdata/module1/topic1.html': '<html><body><a href="module1/index.md">Module 1</a></body></html>'
+		'/testdata/module1/topic1.html': '<html><body><a href="module1/index.html">Module 1</a></body></html>'
```

Comment:
The markdown would be linking to `.md` does `marked` fix this?

Reply:
I think it works.  The original content, whether it's using .md or .html (or a mix of the two) should just be linking to other files without assuming any processing is happening.  When the link rewriter processes the links it connects them properly - it's storing the original names of the files in its manifest.
The API is changing the file names from .md to .html and the mimetype is based on that, so I think the links resolve, the extension is correct, and the mimetype is correct.  The dry run works but I'll double check the logic too.